File: atcoder.jp/contests/abs/tasks/abc086_a/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];
const print = (...messages: any) => console.log(...messages);

const isOdd = (n: number): boolean => n % 2 === 1;

const inputNumberArray = (): number[] => input().split(" ").map(Number);

const main = () => {
  const [A, B] = inputNumberArray();

  const ans = isOdd(A * B) ? "Odd" : "Even";
  print(ans);
};

main();

File: atcoder.jp/contests/abs/tasks/abc086_a/main.py
def is_odd(n: int) -> bool:
    return n % 2 == 1


def input_int_list() -> "list[int]":
    return list(map(int, input().split()))


def main() -> None:
    A, B = input_int_list()

    ans = "Odd" if is_odd(A * B) else "Even"
    print(ans)


main()
